api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -192,6 +192,18 @@ type AzureManagedControlPlaneSpec struct {
 	// - ChinaCloud: "AzureChinaCloud"
 	// - PublicCloud: "AzurePublicCloud"
 	// - USGovernmentCloud: "AzureUSGovernmentCloud"
+	//
```

Comment:
This type being mostly moved into types_class.go as part of the managed cluster Cluster Class refactoring caused the conflict.